pkg/certificate/controller.go

```diff
@@ -89,6 +89,7 @@ func (m *Manager) Reconcile(request reconcile.Request) (reconcile.Result, error)
 	reqLogger.Info("Reconciling Certificates")
 
 	elapsedToRotate := m.elapsedToRotateFromLastDeadline()
+	elapsedToRotateServices := m.elapsedToRotateServicesFromLastDeadline()
 
 	// Ensure that this Reconcile is not called after bad changes at
 	// the certificate chain
```

Comment:
why do we only consider `if elapsedToRotate > 0` to check TLS verify? shouldn't we also account for the elapsedToRotateServices?

Reply:
I recline my comment :)